pkgs/by-name/zs/zscaler-cacert/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "zscaler-cacert";
+
+  # check it on the DHL keyserver homepage ('validity period')
+  # and using openssl:
+  # openssl x509 -startdate -noout -in ZscalerRootCertificate-2048-SHA256.crt
+
+  version = "0-unstable-2025-02-02";
+
+  src = fetchurl {
+    url = "https://keyserver.dhl.com/pki/X3/ZscalerRootCertificate-2048-SHA256.crt";
+    hash = "sha256-HrQ7am1IteSP1O3mz3weay1NQG8aj3EUt1ZjYRbOEQA=";
+  };
+
+  dontUnpack = true;
+
+  dontBuild = true;
+
+  installPhase = ''
+    install -Dm644 $src $out/etc/ssl/certs/zscaler-ca.crt
+  '';
+
+  meta = {
+    description = "ZScaler Root CA certificate";
+    homepage = "https://keyserver.dhl.com/certificates";
+    license = lib.licenses.cc0;
```

The CC0 Public Domain Dedication is not equivalent to something being in the public domain, as it contains additional fallback grants not present here.

I agree this is in the public domain in any country I'm aware of.

```suggestion
    license = lib.licenses.publicDomain;
```